### PR TITLE
process indicator

### PR DESCRIPTION
**Pull request by nathanyoung**
Adds process indicator markup and documentation to the Components section

### CODE REVIEW

**Comment by nathanyoung** [comment]
screen shot 2016-11-11 at 3 03 41 pm

File: www/views/partials/components/process.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          </a>
+        </li>
+        <li>
+          <a href="#" aria-selected="false" role="tab">
+            <span class="process-step"></span>
+            <span class="process-label truncate" title="billing">billing</span>
+          </a>
+        </li>
+        <li>
+          <a href="#" aria-selected="false" role="tab">
```
Comment (osbornm) [inline www/views/partials/components/process.html]: is `tab` the correct role?

File: www/views/partials/components/process.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            <span class="process-step"></span>
+            <span class="process-label truncate" title="feedback">feedback</span>
+          </a>
+        </li>
+      </ol>
+      <div class="process-progress">
+        <div class="process-progress-bar" style="width:50%"></div>
```
Comment (osbornm) [inline www/views/partials/components/process.html]: remove inline style

File: src/less/process.less [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+.process-step {
+	width: 1.5rem;
+	height: 1.5rem;
+	border-radius: 50%;
+	background-color: @process-step-bg;
+	display: inline-block;
+	vertical-align: middle;
+	position: relative;
+	z-index: 1;
+	transition: e('background-color .2s');
```
Comment (nathanyoung) [inline src/less/process.less]: if you look at the file, it's indented. not sure why it's not rendering properly here.

File: www/views/partials/components/process.html (hunk 1 above)
Comment (nathanyoung) [inline www/views/partials/components/process.html]: yes, it's a `tab` inside a `tablist` set on the `<ol>`

**Pull request merged**